src/pudl/helpers.py

```diff
@@ -292,7 +292,8 @@ def _clean_area_name_col(area_name_col: pd.Series, replace_dict: dict):
     counties = counties[~city_county_dupe_mask]
 
     df = (
-        df.assign(county_tmp=_clean_area_name_col(df[county_col], {}))
+        df.astype({county_col: pd.StringDtype()})
+        .assign(county_tmp=lambda x: _clean_area_name_col(x[county_col], {}))
```

Comment:
Checking understanding: this change is for two reasons:

1. we needed `county_col` as a string, and sometimes it wasn't
2. `_clean_area_name_col` couldn't handle a Series so we have to run it one row at a time

if yes, all good, if no, help?

Reply:
I thought assign with a lambda function / any callable that takes a series still does its work vectorized. Is that not true?